README.md

```diff
@@ -389,12 +389,10 @@ Example request body:
 
 ```source-json
 {
-  "article": {
```

Comment:
the article key was removed. Update the readme

Reply:
I have done that @IssaIan

Reply:
Hello @IssaIan ,
It is already updated..it has been deleted hence the pink colour